examples/registration_form/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+analyzer:
+  plugins:
+    angular:
+      enabled: true
```

Comment:
Should this have a newline? Here and other files?

Reply:
Done